resources/css/style.css

```diff
@@ -1,42 +1,64 @@
+html {
+    overflow: hidden;
+}
+
+/* BACKGROUND SHADOWING */
+
 .opacity {
     width: 100vw;
     height: 100vh;
-    background: hsla(149, 0%, 14%, 0.4);
-    z-index: 10;
+    background: hsla(149, 0%, 14%, 0.5);
 }
 
+/* BACGROUND */
+
 .background {
     width: 100vw;
     height: 100vh;
     background: url('../images/background/table.jpg');
     background-position: 0;
     background-repeat: no-repeat;
-    z-index: 1;
 }
 
+/* PLAYERS POSITIONING */
+
 .player-pair {
     display: flex;
     justify-content: space-between;
     margin: 0 5rem 0 5rem;
 }
 
 .player {
-    margin: 0 0 15rem 0;
+    margin: 1rem 0 0 0;
     padding-top: 0.5rem;
 }
 
-.card {
-    width: 140px;
-    height: 140px;
+.moved {
+    padding: 0 15rem;
+}
+
+/* PLAYERS' PORTAITS SETUP */
+
+.portait {
+    width: 141px;
+    height: 141px;
     background: url("../images/characters/cropped/pedro_cropped.jpg") no-repeat;
```

Comment:
I think for a more flexible solution to link with js we need to pass images as `<img src>`

for example, for each hero, we load their cards via `<img src={playerClass.name}.png />`